docs/source/agent-framework/web-framework.rst

```diff
@@ -14,18 +14,50 @@ Web SubSystem
 Enabling
 --------
 
-The web subsystem is not enabled by default as it is only required by a small subset of agents.  To enable the web subsystem the platform instance must have an enabled the web server and the agent must pass enable_web=True to the agent constructor.
+The web subsystem is not enabled by default as it is only required by a small subset of agents.
+To enable the web subsystem the platform instance must have an enabled the web server and the agent
+must pass enable_web=True to the agent constructor.
 
-Methods
+.. code-block:: python
+
+    class WebAgent(Agent):
+        def __init__(self, **kwargs):
+            super(WebAgent, self).__init__(enable_web=True,**kwargs)
+
+
+MANIFEST File
+-------------
+
+The MANIFEST.in file is used to package additional files needed for your web enabled agent.
+Please read the python packaging `documentation <https://packaging.python.org/guides/using-manifest-in/>`_
+on the MANIFEST.in file. For most cases, i.e. when you only need to include a webroot directory for html
+and javascript, the manifest file only needs to include the `recursive-include` command. For example, the entirety
+of the VolttronCentral MANIFEST.in file is:
+
+.. code-block:: python
+
+    recursive-include volttroncentral/webroot *
+
+The MANIFEST.in file should be located in the root directory of the agent. All pathing for the MANIFEST.in file
+commands are relative to this root directory.
+
+Routes
 -------
 
-The web subsystem allows an agent to register three different types of endpoints; path based, JSON and websocket.  A path based endpoint allows the agent to specify a prefix and a static path on the file system to serve static files.  The prefix can be a regular expression.
+The web subsystem allows an agent to register three different types of routes; file paths, endpoints, and websockets.
+
+.. note::
+    For all routes the first match wins.  Therefore ordering which routes are registered first becomes important.
+
 
-.. note:: The web subsystem is only available when the constructor contains enable_web=True.
+File Path
+~~~~~~~~~
+
+A path based endpoint allows the agent to specify a prefix and a static path on the file system to serve static files.
```

Comment:
Note that a relative path should point to the agent-data directory within the installed agent (which is better than the example you give I think)

Also note that the user MUST have read permissions on the folder